src/components/breadcrumb/breadcrumb.html

```diff
@@ -8,5 +8,8 @@
   <div class="bx--breadcrumb-item">
     <a href="#" class="bx--link">Breadcrumb 3</a>
   </div>
+  <div class="bx--breadcrumb-item bx--breadcrumb-item--last">
+    <span>Breadcrumb 4</span>
```

Comment:
*(Just as a note for myself, no action needed for this PR):*

Need to add typography class for any child elements of `bx--breadcrumb-item` to ensure that it's always going to use `font-size: 14px`.